src/utils/ClickAwayListener.js

```diff
@@ -35,7 +35,7 @@ class ClickAwayListener extends React.Component {
       const el = findDOMNode(this);
 
       if (
-        event.target instanceof HTMLElement &&
+        (event.target instanceof HTMLElement || event.target instanceof SVGElement) &&
```

Comment:
I believe this line was added to please flow type. We have remove flowtype since then. I think that it's safe to remove it.